README.md

```diff
@@ -135,6 +138,18 @@ You can also enqueue jobs without starting workers:
 {:ok, ack} = Exq.Enqueuer.enqueue(:exq_enqueuer, "default", "MyWorker", [])
 
 ```
+You can also schedule jobs to start at a future time:
+You need to make scheduler_enable is set to true
+
+Schedule a job to start in 5 mins
+```elixir
+{:ok, ack} = Exq.enqueue_in(:exq, "default", 300, "MyWorker", ["arg1", "arg2"])
```

Comment:
I'm confused ;) it seems to be a bug in Timex but

enqueue_in works for me in my app with Timex 0.13.5
enqueue_in doesn't work in tests with Timex 0.13.5
enqueue_in does work in tests with Timex 0.16.2 but the performance test now takes 50 secs instead of 10 secs